draft-ietf-emu-eap-tls13.xml

```diff
@@ -257,10 +257,11 @@ href='http://xml.resource.org/authoring/rfc2629.xslt' ?>
 			</list>
 			</t>
 			
-			<t>Figures <xref target="figterm1" format="counter"/>, <xref target="figterm2" format="counter"/>, and <xref target="figterm3" format="counter"/> illustrate message flows in several cases where the EAP-TLS peer or EAP-TLS server sends a TLS fatal alert message.
+			<t>Figures <xref target="figterm1" format="counter"/>, <xref target="figterm2" format="counter"/>, and <xref target="figterm3" format="counter"/> illustrate message flows in several cases where the EAP-TLS peer or EAP-TLS server sends a TLS Error alert message.
 				
+In earlier versions of TLS, Error alerts could be warnings or fatal. In TLS 1.3, Error alerts are always fatal and the only alerts sent at warning level are "close_notify" and "user_cancelled", both of which indicate that the connection is not going to continue normally, see <xref target="RFC8446"/>.</t>
 				
-In earlier versions of TLS, error alerts could be warnings or fatal. In TLS 1.3, error alerts are always fatal and the only alerts sent at warning level are "close_notify" and "user_cancelled", both of which indicate that the connection is not going to continue normally, see <xref target="RFC8446"/>. </t>
+<t>In TLS 1.3 <xref target="RFC5216"/>, Error alerts are not mandatory to send after a fatal error condition. Failure to send TLS Error alerts means that the peer or server would have no way of determining what went wrong. EAP-TLS 1.3 strengthen this requirement. Whenever an implementation encounters a fatal error condition, it MUST send an appropriate TLS Error alert.</t>
```

Comment:
I think we need to be clear on user_cancelled behavior in this section.

Reply:
Yes, definitly. But unless the clarification is MUST NOT send user_cancelled, the clarification might depend on the conclusion from the protected result discussion as well as the TLS discussions on when and how to send alerts. RFC 8446 contradict itself regarding closure alerts.

Reply:
I think the working group is reaching consensus on using application data for result indications.